src/app/(pages)/[slug]/page.tsx

```diff
@@ -18,8 +19,9 @@ export default async function Page() {
     TalksAndRoundtables: await fetchAllContentByType("talks-and-roundtables"),
   };
 
-
   const highlights = await fetchGlobals("homepage-settings", 3);
+  const categories = await fetchAllContentByType('categories').then((res: Category[]) => res.map((item: Category) => item.title))
+
```

Comment:
pre-fetching categories to feed searchbox and facilitate filtering instead of forcing async on SearchBox component 
(also avoids conflicts between `use server` and `use client`